homeassistant/components/modbus/climate.py

```diff
@@ -70,6 +77,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     modbus_slave = config.get(CONF_SLAVE)
     target_temp_register = config.get(CONF_TARGET_TEMP)
     current_temp_register = config.get(CONF_CURRENT_TEMP)
+    current_temp_register_type = config.get(CONF_CURRENT_TEMP_REGISTER_TYPE)
```

Comment:
Please use `dict[key]` for all optional config keys that have a default value.

Reply:
Done.